src/modules/faucet/FaucetAssetsList.tsx

```diff
@@ -132,9 +132,17 @@ export default function FaucetAssetsList() {
             )}
 
             <ListColumn maxWidth={280} align="right">
-              <Button variant="contained" onClick={() => openFaucet(reserve.underlyingAsset)}>
-                <Trans>Faucet</Trans>
-              </Button>
+              {!currentMarketData.enabledFeatures ? (
+                <Link href={`https://faucet.circle.com/`}>
```

Comment:
Having a button inside the link component works, but could cause some issues. Instead, you can have the button rendered as an anchor tag using by setting the component prop to be a link (also added in an external link icon so it's clear that it will open to an external link):
```suggestion
                <Button
                    href="https://faucet.circle.com/"
                    component={Link}
                    variant="contained"
                    endIcon={
                      <SvgIcon sx={{ width: 14, height: 14 }}>
                        <ExternalLinkIcon />
                      </SvgIcon>
                    }
                  >
                    <Trans>Faucet</Trans>
                  </Button>
```